Docker/Dockerfile

```diff
@@ -7,45 +7,46 @@ ENV DEBIAN_FRONTEND=noninteractive
 
 # Install custom libraries + freesurfer 6.0 (and necessary dependencies)
 RUN apt-get update && apt-get install -y --no-install-recommends \
-         build-essential \
-         cmake \
-         git \
-         vim \
-         wget \
-         ca-certificates \
```

Comment:
We may need to include `ca-certificates` again, due to the aforementioned build error.